src/set_theory/game.lean

```diff
@@ -251,33 +264,37 @@ rfl
    = x.move_right i * y + x * y.move_left j - x.move_right i * y.move_left j :=
 by {cases x, cases y, refl}
 
-/-- `x * y` has exactly the same moves as `y * x`. -/
-def mul_comm_relabelling (x y : pgame.{u}) : (x * y).relabelling (y * x) :=
+theorem quot_mul_comm : Π (x y : pgame.{u}), ⟦x * y⟧ = ⟦y * x⟧
+| (mk xl xr xL xR) (mk yl yr yL yR) :=
```

Comment:
Is this actually an improvement? Or is it just to make things more uniform? (That's an okay reason to change it, I suppose!)

Reply:
I was unsure about this one too. My reasoning is that the proofs are easier to read so it's worth the change. 

Instead of statements like
``` lean
 { exact ((add_comm_relabelling _ _).trans
                ((IHyl j).add_congr (IHxl i y))).sub_congr (IHxl i (yL j)) },
```

we get 
```lean
{ change ⟦xL i * y⟧ + ⟦x * yL j⟧ - ⟦xL i * yL j⟧ = ⟦yL j * x⟧ + ⟦y * xL i⟧ - ⟦yL j * xL i⟧,
      rw [quot_mul_comm (xL i) y, quot_mul_comm x (yL j), quot_mul_comm (xL i) (yL j)],```

But the actual proof statement is definitely weaker (although the only use of `relabelling` seems to be to prove `equivalence`.

I'd be happy to revert this if a stronger statement is preferable.

Reply:
No, let's leave as is.